components/http-delivery.js

```diff
@@ -0,0 +1,19 @@
+// http-delivery.js
+
+var _ = require('underscore');
+
+var wrapper = require('../src/javascript-wrapper.js');
+
+/**
+ * This updater takes response content, response type, and a {req, res} pair as
+ * input. It then writes the content and type to the response and returns the
+ * input pair.
+ */
+module.exports = wrapper(function(content, type, input) {
```

Comment:
Is it a common convention to call the {req, res} pair 'input'?  If not it would seem clearer to me to call it reqRes or something like that.